proto/gen/api/v1/api.swagger.json

```diff
@@ -1349,13 +1349,14 @@
       "enum": [
         "TYPE_UNSPECIFIED",
         "TYPE_STRING",
-        "TYPE_NUMBER",
+        "TYPE_INT",
```

Comment:
Please also execute `make generate`, the swagger file will then be copied over into `pkg/web/openapi/swagger-ui/api/v1/api.swagger.json`.